patientInfo/src/main/java/fr/marc/patientInfo/serviceImpl/PatientInfoServiceImpl.java

```diff
@@ -99,7 +96,7 @@ public Patient updatePatient(Patient patient) {
 	}
 
 	@Override
-	public Patient createPatient(Patient patient) {
+	public Patient createPatient(Patient patient){
 		// Test if the patient to update already exist
 		Optional<Patient> findingPatient = patientInfoRepository.findFirstByFamilyAndGivenAndDob(patient.getFamily(),patient.getGiven(), patient.getDob());
 		if (findingPatient.isPresent()&&!findingPatient.get().getId().equals(patient.getId())) {
```

Comment:
nit: (missing blanck spaces)

```java
(findingPatient.isPresent() && !findingPatient.get().getId().equals(patient.getId()))
```

Reply:
Si la question est sur l'espace en trop, il n'y est pas sur mon IDE (Je les avais enlevé suite à tes remarques. Par contre, mon IDE m'en ajoute automatiquement lors de la création d'un méthode...)
Si la question est sur (findingPatient.isPresent() && !findingPatient.get().getId().equals(patient.getId())), j'ai ajouté ces 2 conditions car, quand on arrive sur la page PatientSearch.html, comme il n'y a pas de patient entré elle me mettait d'office le message d'erreur "Patient not found". Donc, quand rien n'est rentré, il n'y a pas ce message. Par contre, il y a 2 message, Last name is mandatory et First name is mandatory.